packages/common/util/src/entry.ts

```diff
@@ -1,10 +1,10 @@
-/**
- * Inspired by https://doc.rust-lang.org/std/collections/struct.HashMap.html#method.entry
- */
 //
 // Copyright 2023 DXOS.org
 //
 
+/**
+ * Inspired by https://doc.rust-lang.org/std/collections/struct.HashMap.html#method.entry
+ */
 export const entry = <K, V>(map: Map<K, V>, key: K): MapEntry<K, V, undefined> => new MapEntry(map, key);
```

Comment:
rather than orInsert, to me it would be clearer to have something like https://docs.python.org/3/library/collections.html#collections.defaultdict

e.g., `const value = createOrInsert(map, key, () => 'default init value')`